FlashDevelop/FlashDevelop.csproj

```diff
@@ -257,13 +257,20 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' != 'Unix' ">
     <PreBuildEvent>"$(SolutionDir)\SetVersion.bat" "$(SolutionDir)"</PreBuildEvent>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Platform)' == 'AnyCPU' ">
+  <PropertyGroup Condition=" '$(OS)' == 'Unix' ">
+    <PreBuildEvent>"$(SolutionDir)\SetVersion.sh" "$(SolutionDir)"</PreBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' != 'Unix' And '$(Platform)' == 'AnyCPU' ">
     <PreBuildEvent>"$(SolutionDir)\SetVersion.bat" "$(SolutionDir)" &amp;&amp; ^
 IF EXIST "$(TargetPath)" move /Y "$(TargetPath)" "$(TargetDir)$(TargetName)86$(TargetExt)"</PreBuildEvent>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Unix' And '$(Platform)' == 'AnyCPU' ">
+    <PreBuildEvent>"$(SolutionDir)\SetVersion.sh" "$(SolutionDir)" &amp;&amp; ^
+IF EXIST "$(TargetPath)" move /Y "$(TargetPath)" "$(TargetDir)$(TargetName)86$(TargetExt)"</PreBuildEvent>
```

Comment:
Travis-CI config https://github.com/SlavaRa/flashdevelop/blob/development-linux/.travis.yml
I can add a build there for a 64-bit system, but this has nothing to do with the configuration of the project, it is made in analogy with the Windows version
I do not understand, what is the problem now?